src/services/email.service.js

```diff
@@ -0,0 +1,69 @@
+require('dotenv/config');
+
+const nodemailer = require('nodemailer');
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASS,
+  },
+});
```

Comment:
Consider adding error handling for the transporter creation to catch any issues with the SMTP configuration.